gramineproject/gramine
# [CI] Ubuntu 24.04 and 22.04 part 6/6

## Description of the changes 

See #2026 for rationale. This PR concludes the series.

## How to test this PR? 

- CI
- To check if I haven't missed anything from the previous PR, `git fetch && git diff origin/woju/ubuntu-noble-3 origin/woju/noble-stack-5` (should be empty).


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gramine/2031)
